certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Status"
+                },
+                "subject": {
+                  "type": "string",
+                  "description": "Subject"
+                },
+                "msg": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "integer",
+                        "description": "Request ID",
+                        "title": "",
+                        "format": "string"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "Request Date"
+                      }
+                    }
+                  },
+                  "description": "msg"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Approval Processes Requests",
+        "description": "Get Approval Process Requests",
+        "operationId": "GetApprovalProcessRequests",
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/get_approval_stages/{approval_process_request_id}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Status"
+                },
+                "subject": {
+                  "type": "string",
+                  "description": "Subject"
+                },
+                "msg": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "Stage ID"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "Stage Name"
+                      }
+                    }
+                  },
+                  "description": "msg"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Process Approval Stages",
+        "description": "Get Process Approval Request Stages",
+        "operationId": "GetProcessApprovalRequestStages",
+        "parameters": [
+          {
+            "name": "approval_process_request_id",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/retrieve_approval_process_request/{approval_process_request_id}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.